packages/ui/scripts/grayscale-audit/font_size_v2.py

```
@@ -0,0 +1,112 @@
+when a font-size question arises. Caller supplies a screenshot, a starting x, and a
+y-band that fully contains the line; the script finds the first ink glyph in that
+band, measures its tight bbox, and reports a font-size estimate based on a typical
+typographic ratio for the given script.
+"""
+import argparse
+import sys
+
+from PIL import Image
+
+
+def find_first_glyph_x(img, y_start, y_end, x_search_start, x_search_end, dark_text=True, threshold=180):
```

Ruff (0.15.12)

[warning] Boolean default positional argument in function definition

(FBT002)